### PR TITLE
🌱 Add more debug logging in binding controller

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR adds a bit more logging in the binding controller.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
Among the things revealed by this new logging:

```
    controller.go:198: I0219 02:01:37.398505] Binding: Ignoring excluded APIGroup groupVersion="flowcontrol.apiserver.k8s.io/v1beta3"
    controller.go:198: I0219 02:01:37.398512] Binding: Ignoring excluded APIGroup groupVersion="flowcontrol.apiserver.k8s.io/v1beta2"
    controller.go:206: I0219 02:01:37.398517] Binding: Working on API Group groupVersion="flowcontrol.apiserver.k8s.io/v1beta1" numResources=0
    controller.go:206: I0219 02:01:37.398523] Binding: Working on API Group groupVersion="flowcontrol.apiserver.k8s.io/v1alpha1" numResources=0
```

**Review by vMaroon — APPROVED** [review]
/lgtm

File: pkg/binding/controller.go [hunk 1]
```diff
@@ -271,7 +275,9 @@ func (c *Controller) run(ctx context.Context, workers int) error {
 
 	c.logger.Info("Starting workers", "count", workers)
 	for i := 0; i < workers; i++ {
-		go wait.UntilWithContext(ctx, c.runWorker, time.Second)
+		logger := c.logger.WithName(fmt.Sprintf("%s-worker-%d", controllerName, i))
```
Comment (nirrozenbaum) [inline pkg/binding/controller.go:278]: why not puting worker id in KeyValue pair? 
not very critical but looks cleaner

Comment (MikeSpreitzer) [reply pkg/binding/controller.go:278]: Good idea. Next rev.

**Comment by pdettori** [comment]
> Among the things revealed by this new logging:

@MikeSpreitzer - good catch, looks like we nee to add more to exclude list or just allow to exclude by group for all versions.

**Pull request merged**